build/cluster.js

```diff
@@ -0,0 +1,218 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Gulp tasks for kubernetes cluster management.
+ */
+import childProcess from 'child_process';
+import del from 'del';
+import gulp from 'gulp';
+import chmod from 'gulp-chmod';
+import gulpFilter from 'gulp-filter';
+import git from 'gulp-git';
+import gunzip from 'gulp-gunzip';
+import untar from 'gulp-untar';
+import gulpUtil from 'gulp-util';
+import pathExists from'path-exists';
+import request from 'request';
+import source from 'vinyl-source-stream';
+
+import conf from './conf';
+
+
+let clusterProcess = null,
+    exec = childProcess.exec,
+    isClusterRunning = false,
+    kubernetesArchive = 'kubernetes.tar.gz',
+    kubernetesUrl = 'https://github.com/kubernetes/kubernetes.git',
+    spawn = childProcess.spawn,
```

Comment:
I dont see a reason we should alias childProcess.spawn. It is used only once. So in the place you use it, you can just write childProcess.spawn :)